tooling/hibernate-maven-plugin/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  
+    <modelVersion>4.0.0</modelVersion>
+     
+    <groupId>org.hibernate.orm</groupId>
+    <artifactId>hibernate-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <version>0.0.1-SNAPSHOT</version>
```

Comment:
This really needs to mirror the version from the Gradle project.  And really quite a few things in this file should as well.  

We had a discussion about whether we could/should leverage `<properties/>` for this or generate this file from the Gradle project.  I don't feel super strongly about either way.  `<properties/>` is obviously an easier solution - it comes down to how we want to handle dependencies.

Reply:
I agree that ideally the pom.xml file should be generated. I left the pom.xml file there for now because it still allows me to build the plugin using Maven. And yes, of course the artifact id and all should be changed and taken from the gradle project.

Reply:
> I agree that ideally the pom.xml file should be generated

That's definitely one option.  I'm not sure its "better" though.  I think it depends on how we want to deal with dependencies.  I assume that we will have 3 sets (at least) of dependencies involved here -

1. On the Gradle side for MavenEmbedder
2. On the Maven side
2.a. for compiling and bundling the plugin
2.b. for integration testing the plugin

I think it comes down to whether (1) and (2) share any dependencies and whether we want to hand-maintain these between the 2 descriptors.  Specifically potential expansion / changes to the set of dependencies.  For "just" versions, that's easy to handle through properties as well.

> the artifact id and all should be changed and taken from the gradle project.

I'm not sure EVERYTHING needs to come from the Gradle project.  I doubt we will be changing the group-id or artifact-id e.g.